.travis/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ "$(uname -s)" == 'Darwin' ]]; then
+    DARWIN=true
+else
+    DARWIN=false
+fi
+
+if [[ "$DARWIN" = true ]]; then
+    brew update
+
+    # xvfb for OS X?
+
+    # flash via command line for OS X?
```

Comment:
You will have to download the DMG and get the package installed. It's not something we need right now, so lets defer flash also for Linux. Please file another bug for it.